tests/e2e/trusted_ips_test.go

```diff
@@ -0,0 +1,54 @@
+package e2e
+
+import (
+	// nolint
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/ci"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/exec"
+)
+
+var _ = Describe("List trusted IPs", func() {
+	defer GinkgoRecover()
+
+	var (
+		err               error
+		trustedIPsService exec.TrustedIPsService
+	)
+
+	BeforeEach(func() {
+		trustedIPsService, err = exec.NewTrustedIPsService(constants.TrustedIPsDir)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("list trusted IPs - [id:75233]", ci.Day1, ci.Medium, func() {
+		By("Get list of trusted IPs")
+		args := exec.TrustedIPsArgs{}
+		err = trustedIPsService.Init()
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
Init is already done when calling `exec.NewTrustedIPsService`, no need to redo it

```suggestion
		err = trustedIPsService.Init()
		Expect(err).ToNot(HaveOccurred())
```